samples/notebooks/prediction/SDK_Custom_Predict_SDK_Integration.ipynb

```diff
@@ -1078,11 +1123,100 @@
     "\n",
     "With the Custom Prediction Routine feature, it is easy to test the container locally.\n",
     "\n",
-    "Note: You need to have the credentials set up in the previous step and pass the path to the credentials while running the container. The service account should have the **Storage Object Viewer** permission.\n",
-    "\n",
     "In this example, the container executes a prediction request and a health check."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Option 1. Pass local paths**\n",
```

Comment:
Since printing the responses and viewing logs are the same for both options, can we structure this section to be...

```
Deploy your CPR container locally and send some requests.

Option 1. Deploy locally using a local path for `artifacts_uri`
[code]
Option 2. Deploy locally using a GCS path for `artifacts_uri`
[code]

Print out the health check response and its content...

Print out the predict response and its content...

Also print out all the container logs...
```

Also, can you change the code snippet to send a health check first, before the predict request?